.github/workflows/semantic-commit-lint.yml

```diff
@@ -1,7 +1,7 @@
 name: 'Semantic Commit Linting of PR titles'
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
This will fix the errors when dependabot is creating PR (this creates a more secure context, that has also write property)